discovery/gce/gce.go

```diff
@@ -178,6 +178,15 @@ func (d *Discovery) refresh(ctx context.Context) ([]*targetgroup.Group, error) {
 			addr := fmt.Sprintf("%s:%d", priIface.NetworkIP, d.port)
 			labels[model.AddressLabel] = model.LabelValue(addr)
 
+			// Append additional interface private IP address metadata if multiple interfaces exist.
+			if len(inst.NetworkInterfaces) > 1 {
+				gceLabelEthAddressPrefix := gceLabel + "private_ip_eth"
```

Comment:
I don't understand calling this "private_ip_...". If there are two, one private, one public, this will name them both "private".

This should probably just be `interface_addr_eth`. Or possibly `ipv4` since there is IPv6 data in the REST API[0].

[0]: https://cloud.google.com/compute/docs/reference/rest/v1/instances

Reply:
updated